edgexfoundry/edgex-ui-go
# build: Update to latest go mods and adjust code as needed

Note: Still using Gorilla/MUX due to issues using Echo. That change will need to be in a separate PR.







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
make build
verify no issues
make docker
verify no issues
Using Compose Builder run
```
 make run no-secty ds-virtual ui-dev ds-onvif-camera ds-usb-camera ds-llrp
```
Verify UI loaded content as before in 3.0 and operates as expected
Provision Watcher Tab has issue what already has a bug files.

## New Dependency Instructions (If applicable)
